setup.py

```diff
@@ -45,9 +45,12 @@ def get_version():
 
 
 extra_requires = {
+    "benchmark": read_requirements("requirements-benchmark.txt"),
+    # Alias to avoid breaking existing requirements.txt files where [common] is used
+    "common": read_requirements("requirements.txt"),
     "dev": read_requirements("requirements-dev.txt"),
-    "jupyter": read_requirements("requirements-jupyter.txt") + read_requirements("requirements-common.txt"),
-    "common": read_requirements("requirements-common.txt"),
+    "jupyter": read_requirements("requirements-jupyter.txt"),
```

Comment:
I think jupyter is a combination of benchmarks and the jupyter requirements?

```suggestion
    "jupyter": read_requirements("requirements-jupyter.txt") + read_requirements("requirements-benchmark.txt"),
```